### PR TITLE
add metrics endpoint documentation

### PR DESCRIPTION
**Pull request by cameran20**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes [#8](https://usxtech.atlassian.net/browse/CLOUD-8)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Sanity Testing

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by hpatel826** [comment]
@cameran20 I'm going to be making some changes to this repo so I am leaving this PR open for now so I remember to incorporate this documentation. Nice write-up on confluence!

File: charts/variant-api/README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ EOF
      - [.NET](https://github.com/prometheus-net/prometheus-net)
      - Node - [NestJS](https://github.com/digikare/nestjs-prom), [Express](https://github.com/joao-fontenele/express-prometheus-middleware)
      - Python - [Flask](https://github.com/rycus86/prometheus_flask_exporter), [Django](https://github.com/korfuri/django-prometheus)
+    - Documentation with some instructions on how to implement a metrics endpoint can be found [here](https://drivevariant.atlassian.net/wiki/spaces/SRE/pages/2353594376/Service+Metrics+Endpoint)
```
Comment (vibindaniel) [inline charts/variant-api/README.md:68]: I've added it at https://drivevariant.atlassian.net/wiki/spaces/CLOUD/pages/2095185929/Deployment+FAQs#%2Fmetrics-endpoint instead.

Comment (vibindaniel) [reply charts/variant-api/README.md:68]: Like @hpatel826 said, more changes will be occurring here with proper helm docs.

**Comment by salyateem13** [comment]
Closing this PR due to inactivity

**Pull request closed without merging**